### DISCUSSION TITLE
i18n: Implementing label [SearchInput] and [CreateCollection] | Translation and adjustments pt-BR

### DISCUSSION THREAD

**Discussion by DiegoRodriguez83** [open-webui/open-webui#6673] [Show and tell]
### Description

Implementing label [SearchInput] and [CreateCollection] | Translation and adjustments pt-BR)

### Changed
**SearchInput**
**CreateCollection**

### Updated

- All translation.json with this properties

**Comment by alvarellos** [comment]
Hola Diego, I have added Support for Multi Language for the Openwebui. It might be interesting for you.

Here is the discussion
[[Amber](https://github.com/amberyufangchiu)](https://github.com/open-webui/open-webui/discussions/15839)
and the PR
https://github.com/open-webui/open-webui/pull/15840

I would appreciate to have a feedback :)

Thanks !!
